Contributing.md

```diff
@@ -0,0 +1,120 @@
+
+This document describes a very simple process suitable for activities
+in the Planetary Software Organization ecosystem, including the TC
+itself.
+
+The goal of this document is to create a contribution process that:
+
+* Encourages new contributions.
+* Encourages contributors to remain involved.
+* Avoids unnecessary processes and bureaucracy whenever possible.
+* Creates a transparent decision making process which makes it clear how
+contributors can be involved in decision making.
+
+
+## Vocabulary
+
+* A **Contributor** is any individual creating or commenting on an
+	issue or pull request.
+* A **Committer** is a subset of contributors who have been given
+	write access to the repository.
+* The **Technical Committee (TC)** is a group of committers representing
+	the required technical expertise to resolve rare disputes.
+
+
+# Logging Issues
+
+Log an issue for any question or problem you might have. When in
+doubt, log an issue, any additional policies about what to include
+will be provided in the responses. The only exception is a security
+disclosure which should be sent privately to the contact address
+listed in the [Code of Conduct](CoC).
+
+Committers may direct you to another repository, ask for additional
+clarifications, and add appropriate metadata before the issue is
+addressed.
+
+Please be courteous and respectful.  Every participant is expected
+to follow the [Code of Conduct](CoC)).
+
+# Contributions
+
+Any change to resources in this repository must be through pull
+requests (PRs). This applies to all changes to documentation, code,
+binary files, etc. Even long term committers and TC members must
+use pull requests.
+
+In general, the submitter of a PR is responsible for making changes
+to the PR.  Any changes to the PR can be suggested by others in the
+PR thread (or via PRs to the PR), but changes to the primary PR
+should be made by the PR author.  In order to merge a PR, it must
+satisfy both of these two conditions:
+
+  1. have been open for 24 hours
+  2. have two approvals
+
+Pull requests should sit for at least 24 hours to ensure that
+contributors in other timezones have time to review. Consideration
+should also be given to weekends and other holiday periods to ensure
+active committers all have reasonable time to become involved in
+the discussion and review process if they wish.
+
+In order to encourage involvement and review, we require at least
+two explicit approvals from committers that are not the PR author.
+
+The default for each contribution is that it is accepted once no
+committer has an objection, and the above two requirements are
+satisfied.  During review committers may also request that a specific
+contributor who is more versed in a particular area gives an approval
+before the PR can be merged. There is no additional "sign off"
+process for contributions to be merged. Once all issues brought by
+committers are addressed it can be merged by any committer.
+
```

Comment:
Even the PR author? I though we excluded that in PSO?

Reply:
If there are two approvals, and for some reason the second approver didn't merge the PR, the PR author (or any committer) could then merge it.  This is identical from the current PSO text.